### PR TITLE
[frontend] Add chinese language option

### PR DESCRIPTION
**Pull request by RomuDeuxfois**

### CODE REVIEW

File: openbas-front/src/utils/Localization.js [hunk 1]
```diff
@@ -1165,46 +1166,46 @@ const i18n = {
...
+      'Content should not be empty': 'Le contenu ne doit pas être vide',
+      Timezone: 'Fuseau horaire',
+      Log: 'Journal',
+      'Your file should be a XLS': 'Votre fichier doit être un XLS',
+      'Launch import': 'Lancer l\'importation',
+      'Show timeline': 'Afficher la chronologie',
+      'Selected payload': 'Charge utile sélectionnée',
+      Prevention: 'Prévention',
+      'Start date should be at least today': 'La date de début doit être au moins aujourd\'hui',
+      'Update the xls mapper': 'Mettre à jour le mapper XLS',
```
Comment (Dimfacion) [inline openbas-front/src/utils/Localization.js:1191]: It hurts me to say this but it looks like the french word for "mapper" is "mappeur".

File: openbas-front/src/admin/components/profile/UserForm.js [hunk 2]
```diff
@@ -96,6 +96,9 @@ const UserForm = ({ onSubmit, initialValues }) => {
             <MenuItem key="fr" value="fr">
               {t('French')}
             </MenuItem>
+            <MenuItem key="zh" value="zh">
+              {t('Chinese')}
```
Comment (Dimfacion) [inline openbas-front/src/admin/components/profile/UserForm.js:100]: The translation is added on the profile but shouldn't we add it to the list of default languages of the platform too ?

Comment (RomuDeuxfois) [reply openbas-front/src/admin/components/profile/UserForm.js:100]: I am not sure to understand your point
It's already available in the AppIntlProvider class

Comment (RomuDeuxfois) [reply openbas-front/src/admin/components/profile/UserForm.js:100]: Oh, I missed the BrownserLanguage and the moment configuration

File: openbas-front/src/utils/Localization.js (hunk 1 above)
Comment (RomuDeuxfois) [reply openbas-front/src/utils/Localization.js:1191]: Aouch good call

**Comment by RomuDeuxfois** [comment]
@Dimfacion ready for a second round

**Comment by Dimfacion** [comment]
Chinese is still missing in the default language dropdown
image

**Comment by RomuDeuxfois** [comment]
> Chinese is still missing in the default language dropdown image

Forget this one..
I fixed it

**Review by savacano28 — APPROVED** [review]

**Pull request merged**